pkg/kubelet/util/manager/watch_based_manager.go

```diff
@@ -388,6 +404,7 @@ func (c *objectCache) shutdownWhenStopped(stopCh <-chan struct{}) {
 //     referenced objects that aren't referenced from other registered pods
 //   - every GetObject() returns a value from local cache propagated via watches
 func NewWatchBasedManager(
+	parentCtx context.Context,
```

Comment:
Why isn't this called simply `ctx`? It is the context of the instance and becomes the parent context somewhere down the line.

Reply:
Sure, changed.